lua/lspconfig/dotls.lua

```diff
@@ -0,0 +1,29 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+local server_name = "dotls"
+local bin_name = "dot-language-server"
+
+configs[server_name] = {
+  default_config = {
+    cmd = {bin_name, "--stdio"};
+    filetypes = {"dot"};
+    root_dir = util.path.dirname;
```

Comment:
I would match what other language servers do, and fallback to the directory of the current file after checking for a parent git repo. You can see pyright for an example of this.

Reply:
Got it. I'll follow the pyright example. Thanks @mjlbach

Reply:
Thank you!